client.go

```diff
@@ -214,7 +225,7 @@ func (s *Client) FetchSwaps(ctx context.Context) ([]*SwapInfo, error) {
 			LastUpdate:    swp.LastUpdateTime(),
 		}
 
-		htlc, err := GetHtlc(
+		htlc, err := utils.GetHtlc(
```

Comment:
Should be in the commit which introduced this change. Ideally also we want all commits to be buildable and all tests green too (unless not possible).

Reply:
bump

Reply:
bump